Tools/autotest/autotest.py

```diff
@@ -220,17 +235,9 @@ def run_step(step):
 
     binary = binary_path(step, debug=opts.debug)
 
-    if step == 'defaults.ArduPlane':
-        return get_default_params('ArduPlane', binary)
-
-    if step == 'defaults.ArduCopter':
-        return get_default_params('ArduCopter', binary)
-
-    if step == 'defaults.APMrover2':
-        return get_default_params('APMrover2', binary)
-    
-    if step == 'defaults.ArduSub':
-        return get_default_params('ArduSub', binary)
+    if step.startswith("default"):
```

Comment:
I can't help but think we should be using `string.split()` here.  Not a blocker.